Lib/ctypes/test/test_bitfields.py

```diff
@@ -233,6 +233,89 @@ class X(Structure):
         else:
             self.assertEqual(sizeof(X), sizeof(c_int) * 2)
 
+    def test_packed_1(self):
+        class X(Structure):
+            _pack_ = 1
+            _fields_ = [("a", c_uint8, 4),
+                        ("b", c_uint8, 4)]
+        if os.name != "nt":
+            self.assertEqual(sizeof(X), 1)
```

Comment:
It would be better to use a `@unittest.skipIf` decorator.

Also, I don't think this patch needs to add 8 new test methods.

Reply:
I don't think we should reduce the number of tests, but they could probably go in a single method with subtests, all under a single `unittest.skipIf` decorator.

Reply:
Yes, that's what I was planning to do, move to subtests. I don't think Zackery was suggesting reducing the tests, just the test methods, for which the obvious solution is to use subtests instead :blush: